todo_app/data/trello_items.py

```diff
@@ -20,18 +45,19 @@ def get_items():
     Returns: response json() containing trello card names / id's and lists id's
     """
 
-    url = 'https://api.trello.com/1/boards/' + boardid +'/cards?fields=name,idList,dateLastActivity'
+    url = 'https://api.trello.com/1/boards/' + str(get_boardid) +'/cards?fields=name,idList,dateLastActivity'
```

Comment:
You're not actually calling this function here, so it's trying to stringify the function itself rather than the result of it - leading to JSON decoding errors when Trello denies know which board you're referring to
```suggestion
    url = 'https://api.trello.com/1/boards/' + str(get_boardid()) +'/cards?fields=name,idList,dateLastActivity'
```